pkg/util/paramtable/service_param.go

```diff
@@ -625,6 +653,122 @@ please adjust in embedded Milvus: /tmp/milvus/rdb_data`,
 	r.TickerTimeInSeconds.Init(base.mgr)
 }
 
+// NatsmqConfig describes the configuration options for the Nats message queue
+type NatsmqConfig struct {
+	Server natsmqConfigServer
+}
+
+type natsmqConfigServer struct {
+	Port              ParamItem
+	StoreDir          ParamItem
+	MaxMemoryStore    ParamItem
+	MaxFileStore      ParamItem
+	MaxPayload        ParamItem
+	MaxPending        ParamItem
+	Monitor           natsmqConfigServerMonitor
+	InitializeTimeout ParamItem
+}
+
+type natsmqConfigServerMonitor struct {
+	Debug        ParamItem
+	LogTime      ParamItem
+	LogFile      ParamItem
+	LogSizeLimit ParamItem
+}
+
+// Init sets up a new NatsmqConfig instance using the provided BaseTable
+func (r *NatsmqConfig) Init(base *BaseTable) {
+	r.Server.Port = ParamItem{
+		Key:          "natsmq.server.port",
+		Version:      "2.2.0",
+		DefaultValue: "4222",
+		Doc:          `Port for nats server listening`,
+		Export:       true,
+	}
+	r.Server.Port.Init(base.mgr)
+	r.Server.StoreDir = ParamItem{
+		Key:          "natsmq.server.storeDir",
+		Version:      "2.2.0",
+		DefaultValue: "/var/lib/milvus/nats",
+		Doc:          `Directory to use for JetStream storage of nats`,
+		Export:       true,
+	}
+	r.Server.StoreDir.Init(base.mgr)
+	r.Server.MaxMemoryStore = ParamItem{
+		Key:          "natsmq.server.maxMemoryStore",
+		Version:      "2.2.0",
+		DefaultValue: "536870912",
```

Comment:
256 MB?

Reply:
new commit changed:
memory storage is removed, it's not used in natsmq.